pkg/distro/fedora/images.go

```diff
@@ -211,6 +211,9 @@ func osCustomizations(
 			osc.Directories = append(osc.Directories, tailoringDir)
 		}
 
+		// add this after tailoring
+		oscapStageOptions.Compression = true
```

Comment:
For the most part, `oscapStageOptions` and `oscapTailoringStageOptions` are the same, but this field breaks things.
I didn't split the definitions, but I was very tempted to here, let me know if this would be preferred, see the below comment for further clarification.

Reply:
Are they that much the same?  I see a few common fields in the stages in osbuild, but there's a bunch that are unique to each, no?
I  think they might have diverged enough that we can write different configs for each stage and avoid confusion.